push/views.py

```diff
@@ -49,6 +60,7 @@ def send_notification(request, *args, **kwargs):
 @api_view(['GET'])
 @permission_classes([AllowAny])
 def fetch_notification_status(request, notification_id=None, *args, **kwargs):
+    logger.info('Fetch Notification Status API called')
```

Comment:
this log is not needed. What's needed is to log when the notification status is not found